cloud/amazon/resources/keypair.go

```diff
@@ -134,7 +135,11 @@ func (r *KeyPair) Apply(actual, expected cloud.Resource, applyCluster *cluster.C
 }
 func (r *KeyPair) Delete(actual cloud.Resource, known *cluster.Cluster) (cloud.Resource, error) {
 	logger.Debug("keypair.Delete")
-	force := false
+	if os.Getenv("KUBICORN_FORCE_DELETE_KEY") != "" {
```

Comment:
in this case if I run it as `export KUBICORN_FORCE_DELETE_KEY=false`, it will still delete the AWS key.